itou/common_apps/organizations/models.py

```diff
@@ -16,7 +16,16 @@ class OrganizationQuerySet(models.QuerySet):
     """
 
     def member_required(self, user):
-        return self.filter(members=user, members__is_active=True)
+        membership_model = self.model.members.through
+        # through_fields contains a tuple like ("company", "user")
+        membership_through_field = self.model.members.rel.through_fields[0]
```

Comment:
Nommer les champs me semblerait plus clair.
```suggestion
        structure_field, _user_field = self.model.members.rel.through_fields
```